tests/test_environ_config.py

```diff
@@ -16,6 +16,7 @@
 
 import attr
 import pytest
+from attr.exceptions import FrozenInstanceError
```

Comment:
lint is failing because of this. running pre-commit should fix it

Reply:
Thanks for pointing out. For some reason missed `pre-commit` presence